### ISSUE TITLE
Serious bugs when trying to start divine might

### ISSUE THREAD

**Issue by Jimmayus** [EdenServer/community#1500]
![Jimmayus_2019 04 07_185742](https://user-images.githubusercontent.com/45926115/55694402-ad01c500-5979-11e9-9af9-7c61e8b09951.png)
![Jimmayus_2019 04 07_185751](https://user-images.githubusercontent.com/45926115/55694417-c1de5880-5979-11e9-97e1-91e7dfc3e585.png)
![Jimmayus_2019 04 07_185822](https://user-images.githubusercontent.com/45926115/55694435-cf93de00-5979-11e9-8bc3-6ad25041c746.png)

### Checklist

- [X] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [X ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ X] I have provided my client version in the details. (type /ver into your game window)

### Details

We are unable to reliably flag Divine Might with our pentaspheres. Divine Might is accessible for the person trading their pentasphere, everybody in the alliance briefly (for 1-3 seconds) has the BCNM icon and the chance to select "Divine Might" for a few seconds, but then several things happen:

1) The message "You cannot enter the battlefield at present. Please wait a little longer." pops up. This happened even the first time that someone tried to spawn the BCNM, and it's not obvious that anybody was doing divine might previous to us that might interfere with this, and

2) The person who trades the pentasphere is teleported to an empty instance of the bcnm. They are, for some reason, able to keep trading their pentasphere to the entrance circle (the entrance circle does not allow exit, instead only allowing for trades).

We were able to successfully start the fight one time and everyone in our alliance was able to enter. The fight itself has several interesting bugs that are outside the scope of this one.

**Comment by tacbox3** [comment]
I was one of the members who tried to trade my orb, but was denied entry. It's not clear if this caused the orbs to be "used up," but after the BCNM server was reset, we were able to use one of the previously-tried (bugged) orbs to enter. So I tend to think that my orb should still have been "fresh" and ready for use following the evening described by Jimmy above.

However, we returned the following week to try to get some more clears for other members. But when I tried to use the orb I assumed was still "Fresh", it would not allow me to trigger the Divine Might fight. Some people speculated that this might be intended behavior, and that only uncleared players should be able to pop the fight.

However, a GM contacted me and indicated that having previously cleared the fight should not impact your ability to trigger the BCNM. Thus, it appears that my orb was bugged, or somehow flagged as "used up," by either:

(1) Attempting to trade and getting the bug described by Jimmy above.; or
(2) Clearing the fight with an Ark Pentasphere in your possession "uses up" that orb, regardless of whether or not it was used to pop the BCNM.

I tend to think that #2 is the source of this bug, as some info is inconsistent with #1, as explained above. This would make it challenging for people wanting to clear more than 1 alliance worth of LS members, as they would need to ensure that the orbs are held by the correct members (i.e., those that are sitting out of the first run).

~ Contrabanned
